### DISCUSSION TITLE
Application compatible with Android Auto

### DISCUSSION THREAD

**Discussion by OUARZA** [Waboodoo/HTTP-Shortcuts#251] [Ideas]
Hello,
Would it be possible to make the application compatible with Android Auto?
This will allow home automation commands to be launched from the car.

Mathieu

**Comment by Waboodoo** [comment]
Hi,
I like the idea, and it's been brought up before by others. I think it would fit well within the vision that I have for this app, albeit it's not the prime usecase.

I'm not familiar with Android Auto, so I'd have to spend some time to read up on that and check all the requirements. I've briefly looked into it in the past and came to the conclusion that it would require more effort and time than I currently have for this project. Therefore, I unfortunately can't make this happen on my own, and would need some support from someone who has more experience.
